tests/test_bridge_domain.rb

```diff
@@ -223,4 +151,41 @@ def test_invalid_bd_create
       BridgeDomain.new('90, 5000-5004,100')
     end
   end
+
```

Comment:
I think it may be good to disable the feature in the `cleanup` method: https://github.com/cisco/cisco-network-node-utils/pull/302/files#diff-65aa1d6fa6431ccd35936e979551261cR26

```
def cleanup
  config_no_warn("no feature vni")
  ...
```